src/mappings/Controller/schema.graphql

```diff
@@ -87,6 +89,13 @@ type ContributionRewardExtParam @entity {
 	rewarder: Bytes!
 }
 
+type SchemeFactoryParam @entity {
+	id: ID!
+	votingMachine: Bytes!
+	voteParams: GenesisProtocolParam!
+	daoFactory: Bytes!
```

Comment:
why daoFactory is needed here ?

Reply:
It's a parameter of the SchemeFactory. It's not necessary I guess but I can't l can't know what will be needed by the UI team and we usually index all parameters like that.